test/blackbox-tests/test-cases/enabled_if/eif-dune-describe-crash.t

```diff
@@ -0,0 +1,32 @@
+  $ cat > dune <<EOF
+  > (executable
+  >  (name test)
+  >  (enabled_if
+  >   (= %{system} macosx)))
```

Comment:
Can you use a different value than macosx? Otherwise the test actually fails on macos

Reply:
Done.